src/ui/Misc/FileTypes.ts

```diff
@@ -40,10 +40,14 @@ export class FileTypes {
     const variableValue = `objecttype_${objecttype}`;
     // Most object types have a set of localized strings in the main dictionary
     var localizedString = l(variableValue);
-
+    // Some strings are sent as `objecttype_[...]` to specify a dictionary to use. If there's no match, try using
+    // the main dictionary by keepingo only the value after `_`.
+    if (localizedString.toLowerCase() == variableValue.toLowerCase()) {
```

Comment:
The function already receives the value in a "clean" format. We don't need to take the transformed value and compute the original value with a regex.